packages/Switch/src/Switch.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import RawButton from "@paprika/raw-button";
+
+import SwitchStyles, { KnobStyled, UnderlayStyled } from "./Switch.styles";
```

Comment:
`switchStyles`, with the lower case first letter is the convention we've been going with.